test/InputOutput/hybrid3dcubedsphere.jl

```diff
@@ -10,6 +10,18 @@ using ClimaCore:
     DataLayouts,
     InputOutput
 
+using ClimaComms
+usempi = get(ENV, "CLIMACORE_DISTRIBUTED", "") == "MPI"
+if usempi
+    using ClimaCommsMPI, MPI
+    const comms_ctx = ClimaCommsMPI.MPICommsContext()
+    ClimaComms.init(comms_ctx)
+    filename = MPI.bcast(tempname(), 0, MPI.COMM_WORLD)
```

Comment:
Can we use `iamroot`. I believe ClimaComms may be using `1` for root!

Reply:
This is calling MPI directly (so is 0-based), instead of ClimaComms (which offsets ranks by 1).

We should add a `broadcast` to ClimaComms (https://github.com/CliMA/ClimaComms.jl/issues/21)